### PR TITLE
Quick and easy test to bump up test coverage to 62%

### PR DESCRIPTION
**Pull request by shefalijoshi**
Addresses #3025

Adds two simple tests to the notebook plugin

### Author Checklist
1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? Y
3. Command line build passes? Y
4. Changes have been smoke-tested? Y
5. Testing instructions included? N/A as circleCI will handle this.

### CODE REVIEW

File: src/plugins/notebook/pluginSpec.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
```
Comment (deeptailor) [inline src/plugins/notebook/pluginSpec.js:2]: bump up to 2020

Comment (shefalijoshi) [reply src/plugins/notebook/pluginSpec.js]: Done.

File: src/plugins/notebook/pluginSpec.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            key: 'testNotebookKey',
+            namespace: ''
+        },
+        type: 'notebook'
+    };
+
+    notebookDefinition.initialize(mockNotebookObject);
+
+    it('defines a notebook object type with the correct key', () => {
+        expect(notebookDefinition.key).toEqual(mockNotebookObject.key);
```
Comment (akhenry) [inline src/plugins/notebook/pluginSpec.js:52]: This doesn't seem correct. Should it be:
``` JavaScript
expect(notebookDefinition.key).toEqual(mockNotebookObject.type);
```
?

Comment (shefalijoshi) [reply src/plugins/notebook/pluginSpec.js:52]: No, it fails if I try to use mockNotebookObject.type

**Comment by shefalijoshi** [comment]
Closing this as I don't think it's needed anymore.

**Pull request closed without merging**